app/store/entities/sampleDocuments/data.js

```diff
@@ -45,9 +45,17 @@ export default {
         "public/samples/Finance/USDC balance sheet.png",
         "public/samples/Finance/USDE balance sheet.png"
       ]
+    },
```

Comment:
Are all of these samples below 4MB in size? If not, it will break the CICD deploy.

Reply:
All these files are less than 250KB. You can also check them [here](https://github.com/awslabs/amazon-textract-document-understanding-solution/tree/update/security/samples/Medical)